pkg/cmd/bootstrap.go

```diff
@@ -165,7 +165,15 @@ func initiateInteractiveMode(io *BootstrapParameters, client *utility.Client, cm
 		io.SealedSecretsService.Namespace = ui.EnterSealedSecretService(&io.SealedSecretsService)
 	}
 	if io.GitOpsRepoURL == "" {
-		io.GitOpsRepoURL = ui.EnterGitRepo()
+		gitopsRepoURL := ui.EnterGitRepo()
+		p, err := url.Parse(gitopsRepoURL)
+		if err != nil {
+			return fmt.Errorf("Invalid URL, err: %v", err)
+		}
+		if p.Host == "" {
+			return fmt.Errorf("could not identify host from %q", gitopsRepoURL)
+		}
+		io.GitOpsRepoURL = p.Host
```

Comment:
Thanks for the PR! Shouldn't the flags `io.GitOpsRepoURL` and `io.ServiceRepoURL` have the complete URL instead of just the host?

Reply:
done thanks !!